shared/actions/json/wallets.json

```diff
@@ -1,37 +1,70 @@
 {
-  "prelude": ["import * as Types from '../constants/types/wallets'"],
+  "prelude": [
+    "import * as Types from '../constants/types/wallets'",
+    "import HiddenString from '../util/hidden-string'"
+  ],
   "actions": {
     "assetsReceived": {
       "_description": "Update our store of assets data",
       "accountID": "Types.AccountID",
-      "assets": "Array<Types.Assets>",
+      "assets": "Array<Types.Assets>"
     },
     "loadAssets": {
       "_description": "Refresh our list of assets for a given account",
-      "accountID": "Types.AccountID",
+      "accountID": "Types.AccountID"
     },
     "loadPayments": {
       "_description": "Refresh our list of payments for a given account",
-      "accountID": "Types.AccountID",
+      "accountID": "Types.AccountID"
     },
     "loadAccounts": {
-      "_description": "Refresh our list of accounts",
+      "_description": "Refresh our list of accounts"
     },
     "paymentsReceived": {
       "_description": "Update our store of payments data",
       "accountID": "Types.AccountID",
-      "payments": "Array<Types.Payment>",
+      "payments": "Array<Types.Payment>"
     },
     "accountsReceived": {
       "_description": "Update our store of account data",
-      "accounts": "Array<Types.Account>",
+      "accounts": "Array<Types.Account>"
     },
     "loadAccounts": {
-      "_description": "Refresh our list of accounts",
+      "_description": "Refresh our list of accounts"
     },
     "selectAccount": {
-      "_description": "Select an account",
+      "_description": "Select an account. Optionally navigate to the account page.",
       "accountID": "Types.AccountID",
+      "show?": "boolean"
+    },
+    "linkExistingAccount": {
+      "_description": "Link an existing Stellar account with this Keybase user.",
+      "name": "string",
+      "secretKey": "HiddenString",
+      "canError": {
+        "name": "string",
+        "secretKey": "HiddenString",
+        "error": "string"
```

Comment:
can the error be sensitive

Reply:
No, also @mlsteele says the service will log them so I'll log them here too.